documentation/adr/0008_continue_to_use_Algolia_as_search_engine.md

```diff
@@ -0,0 +1,64 @@
+# Continue to use Algolia as search engine
+
+**Date: 2020-11-16**
+
+## Status
+
+**Decided**
+
+## Parties Involved
+
+ * Alex Bowen
+ * Ben Mitchell
+ * Cesidio Di Landa
+ * Christian Sutter
+ * Chris Taylor
+ * Connor McQuillan
+ * David Mears
+ * Davide Dippolito
+ * Joe Hull
+ * Mili Malde
+
+## Context
+
+The current technical implementation of the Teaching Vacancies web application adopts Algolia as Search Engine to serve job search results to teachers. The development team migrated from Elastic Search to Algolia in April 2020. The reasons behind the migration are outlined in this document:
+
+https://docs.google.com/document/d/1SjmXcnjJyAuAE8WRPrGX7MRrn6UQAfOQyb9GjGsa43c/edit?usp=sharing
+
+
+
+A deeper analysis of TVS search requirements can be found in Search Analysis document:
+
+https://docs.google.com/document/d/1fEQINBHTjI-TqcDfZ6IEDpNAghRGsnQg0EGJoD4E5wg/edit?usp=sharing
+
+Teaching Vacancies conducted a review into which search engine option to take forward in April 2020 and Algolia was compared to Elastic Search. Despite limited technical justification, Algolia was selected as the preferred option to take forward, and the switch to Algolia standard plan was completed on the 18th of May 2020. Since that time a number of features closely linked to search have been introduced, such as job alerts.
```

Comment:
Job alerts were around long before Algolia was introduced

Reply:
@cpjmcquillan We improved the feature though. If it's confusing I'll reword it

Reply:
I would just say something like `Since that time a number of features and functionalities closely linked to search have been introduced and improved` and remove the reference to job alerts